### DISCUSSION TITLE
JavaScript Version

### DISCUSSION THREAD

**Discussion by LOTSOFPYRO** [cwendt94/espn-api#526] [ESPN API]
I'm looking to use an API for Blogs on my Wix Website and I'm being told that Wix runs off JavaScript.

I'm not the best when it comes to coding and I'm looking for any and all help I can get.

**Comment by cwendt94** [comment]
Im not familiar with Wix and how much flexibility you have with updating that type of website. To use this package you would need to have a backend API that your website could call and get the needed information about your league. There could be an espn api javascript packages out there, but it might not be built for the frontend.
